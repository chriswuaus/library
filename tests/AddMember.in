ADD MEMBER Jeremy 
member 100000
add member yeehaw
member 100001
add member sdkfns 
member 100002
add member yeeeeeehaw
member 100003
EXIT
