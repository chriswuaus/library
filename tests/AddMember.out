user: Success.

user: 100000: Jeremy

user: Success.

user: 100001: yeehaw

user: Success.

user: 100002: sdkfns

user: Success.

user: 100003: yeeeeeehaw

user: Ending Library process.
